doc/content/verification_and_validation/ver-1fb.md

```diff
@@ -2,49 +2,46 @@
 
 # Thermal Transient in a Slab
 
-This verification problem is taken from [!cite](ambrosek2008verification). In this problem thermal transient in a slab is modeled. The ends of a slab are kept fixed at different temperatures. The temperature distribution in the slab evolves from an initial state to steady-state. The analytical solution for this case is given as:
+## General Case Description
 
-\begin{equation}
-T(x,t) = T_o \;+\; (T_1-T_o)\Bigg\{1-\frac{x}{L}-\frac{2}{L}\sum_{m=1}^{\infty} \left(\frac{1}{\lambda_m}  \sin(\lambda_m x) \exp(-\alpha \lambda_m^2 t)  \right)\Bigg\}
-\end{equation}
-
-where:
-
-
-    $T$ : temperature in the slab (K)
+This heat transfer verification problem is taken from [!cite](longhurst1992verification) and [!cite](ambrosek2008verification) and builds on the capabilities verified in [ver-1fa](ver-1fa.md). The configuration is the same as in [ver-1fa](ver-1fa.md), except that, in the current case, there are no heat source in the slab and both ends with fixed temperature. This case is simulated in [/ver-1fb.i].
 
+The heat conduction in the one-dimensional model is described as:
 
-    $x$ : distance across the slab (m)
-
-    $t$ : time (seconds)
+\begin{equation} \label{eq:thermal_equation}
+\rho C_P \frac{d T}{d t} = \nabla k \nabla T,
+\end{equation}
 
-    $T_o$ : fixed temperature at one end of the slab (400 K)
+where $T$ is the temperature, $\rho$ is the density, $C_P$ is the specific heat, and $k$ is the thermal conductivity.
 
-    $T_1$ : fixed temperature at the other end of the slab (300 K)
+The ends of a slab are kept fixed at different temperatures. The temperature distribution in the slab evolves from an initial state to steady-state.
 
-    $L$ : length of the slab (4.0 m)
+In this case, the thickness of slab, $L$, is 4.0 m, the thermal conductivity is 1 W/m/K, and the production of material density and specific heat is assumed to be 1 J/m$^3$/K.. The fixed surface temperature, $T_0$ and $T_1$, on both ends are defined as 400 K and 300 K, respectively.
 
-    $\lambda_m$ : $\frac{m\pi}{L}$
+## Analytical solution
 
-    $\alpha$ : thermal diffusivity (1.0 m$^2$/s) where
+[!cite](longhurst1992verification) and [!cite](ambrosek2008verification) provide the analytical solution for the temperature of this case as:
 
-\begin{equation}
-\alpha = \frac{k}{\rho C_p}
+\begin{equation} \label{eq:thermal_analytical}
+T(x,t) = T_0 \;+\; (T_1-T_0)\Bigg\{1-\frac{x}{L}-\frac{2}{L}\sum_{m=1}^{\infty} \left(\frac{1}{\lambda_m}  \sin(\lambda_m x) \exp(-\alpha \lambda_m^2 t)  \right)\Bigg\},
```

Comment:
Please make a note that the expressions provided in TMAP4 and TMAP7 reports seem different, and specify (a) if they are equivalent, or, if not, (b) which one, if any, is correct.